.travis.yml

```diff
@@ -51,6 +51,10 @@ script:
 - cd ..
 - cp travis/dfhack_travis.init "$DF_FOLDER"/
 - python travis/run-tests.py "$DF_FOLDER"
+- cd travis/df-luacheck
+- npm install
```

Comment:
Should probably be in some pre-build step if possible.

Reply:
It requires the `codegen.out.xml` file, and I'm thinking of making a plugin that generates the `builtin.js` file.